main.tf

```diff
@@ -3,9 +3,14 @@ data "azurerm_resource_group" "azlb" {
   name = var.resource_group_name
 }
 
+locals {
+  lb_name  = var.name != "" ? var.name : format("%s-lb", var.prefix)
+  pip_name = var.name != "" ? format("pip-%s", var.name) : format("%s-publicIP", var.prefix)
```

Comment:
Instead of setting the name to "pip-${var.name}", would you create another variable named "pip_name" for this?

Reply:
Done